src/components/DownloadButton.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Button } from 'antd';
+
+const META_TYPE = 'text/json';
+
+function ExportButton({ ast }) {
```

Comment:
consistent naming

```suggestion
function DownloadButton({ ast }) {
```

Reply:
wow this suggestion feature on github looks cool

Reply:
yes. too bad shopee don't want use github.